Documentation/rust/quick-start.rst

```diff
@@ -6,6 +6,9 @@ Quick Start
 This document describes how to get started with kernel development in Rust.
 If you have worked previously with Rust, this will only take a moment.
 
+Please note that, at the moment, a very restricted subset of architectures
+is supported, see :ref:`Documentation/rust/arch-support.rst <_rust_arch_support>`.
```

Comment:
```suggestion
is supported, see :doc:`/rust/arch-support.rst`.
```

Reply:
+1

Reply:
It is:

    :doc:`Documentation/rust/arch-support.rst`

no? Otherwise I get a warning on `htmldocs`.

Reply:
I don't think so, none of the other `:doc:` links I see in there have the `Documentation/` prefix.

Reply:
Ah, I saw the problem: one needs to remove the extension:

    :doc:`/rust/arch-support`.